pkg/kudoctl/kudoinit/migration/migration_helper.go

```diff
@@ -0,0 +1,56 @@
+// This package contains helper functions that can be reused by all migrations
+package migration
+
+import (
+	"context"
+	"fmt"
+
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	kudoapi "github.com/kudobuilder/kudo/pkg/apis/kudo/v1beta1"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/kube"
+)
+
+// ForEachNamespace calls the given function for all namespaces in the cluster
+func ForEachNamespace(client *kube.Client, f func(ns string) error) error {
```

Comment:
Nit: These functions would make sense in `pkg/kudoctl/util/kudo` as well. But with a `kube.Client` and `kudo.Client`, it looks like our usage of the Kubernetes client API will need some refactoring first.

Reply:
Yeaaaah.... The usage of the different clients is something that really should be refactored and cleaned up. Same with the loggers. 

We should have established rules which client(s) and which logger(s) is used where, (i.e. CLI code, Manager code and shared code), but that's a discussion for a different time :)